aepp_middleware/settings.py

```diff
@@ -138,3 +140,18 @@
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+CONSTANCE_CONFIG = {
+    'FAUCET_HOURLY_LIMIT': (50, 'Hour limit of the Faucet'),
+    'FAUCET_DAILY_LIMIT': (500, 'Day limit of the Faucet')
+}
+
+CONSTANCE_REDIS_CONNECTION = {
+    'host': os.getenv('REDIS_HOST', 'redis'),
+    'port': 6379,
+    'db': 1,
+}
```

Comment:
I would leave out redis and constance. It just complicates things. If you want to make changes, you need to restart the server.